docs/deployment/resource-providers/standalone/index.md

```diff
@@ -24,153 +24,203 @@ specific language governing permissions and limitations
- * the amount of available memory per TaskManager (`taskmanager.memory.process.size` and check [memory setup guide]({% link deployment/memory/mem_tuning.md %}#configure-memory-for-standalone-deployment)),
- * the number of available CPUs per machine (`taskmanager.numberOfTaskSlots`),
- * the total number of CPUs in the cluster (`parallelism.default`) and
- * the temporary directories (`io.tmp.dirs`)
+## Standalone Cluster Reference
 
-are very important configuration values.
+### Configuration
 
-{% top %}
+All available configuration options are listed on the [configuration page]({% link deployment/config.md %}), in particular the [Basic Setup]({% link deployment/config.md %}#basic-setup) section contains good advise on configuring the ports, memory, parallelism etc.
+
+### Debugging
+
+If Flink is behaving unexpectedly, we recommend looking at Flink's log files as a starting point for further investigations.
+
+The log files are located in the `logs/` directory. There's a `.log` file for each Flink service running on this machine.
+
+Alternatively, logs are available from the Flink web frontend (both for the JobManager and each TaskManager).
+
+By default, Flink is logging on the "INFO" log level, which provides basic information for all obvious issues. For cases where Flink supposedly behaving wrongly, reducing the log level to "DEBUG" is advised. The logging level is controlled via the `conf/log4.properties` file.
+Setting `rootLogger.level = DEBUG` will boostrap Flink on the DEBUG log level. Note that a restart of Flink is required for the changes to take effect.
 
-### Starting Flink
+There's a dedicated page on the [logging]({%link deployment/advanced/logging.md %}) in Flink.
 
-The following script starts a JobManager on the local node and connects via SSH to all worker nodes listed in the *workers* file to start the TaskManager on each node. Now your Flink system is up and running. The JobManager running on the local node will now accept jobs at the configured RPC port.
+### The start and stop scripts
 
-Assuming that you are on the master node and inside the Flink directory:
+#### start-cluster.sh
 
+The scripts provided with the standalone mode (in the `bin/` directory) use the `conf/workers` and `conf/masters` files, to determine the number of cluster instances to start and stop with the `bin/start-cluster.sh` and `bin/stop-cluster.sh` scripts.
+
+If password-less ssh access to the listed machines is configured, and they share the same directory structure, the script also supports starting and stopping instances remotely.
+
+**Example 1: Start a cluster with 2 TaskManagers locally**
+
+`conf/masters` contents:
 {% highlight bash %}
-bin/start-cluster.sh
+localhost
 {% endhighlight %}
 
-To stop Flink, there is also a `stop-cluster.sh` script.
+`conf/workers` contents:
+{% highlight bash %}
+localhost
+localhost
+{% endhighlight %}
 
-{% top %}
+**Example 2: Start a distributed cluster JobMangers**
```

Comment:
I would consider setting up a distributed standalone cluster a fairly advanced topic. Most of our users are probably using K8s or YARN for deployment.
I'm concerned that this will confuse new users.